### PR TITLE
feat: agreement urls

### PR DESCRIPTION
**Pull request by k1rill**
Configuration fields added:

```
AGREEMENT_URLS:
  PRIVACY_POLICY_URL: ''
  COOKIE_POLICY_URL: ''
  DATA_SELL_CONSENT_URL: ''
  TOS_URL: ''
  EULA_URL: ''
  SUPPORTED_LANGUAGES: [ ] #en is defalut language
```

For `SUPPORTED_LANGUAGES` we are adding lang to the url path

If configuration field is empty, button will be hided in profile.

https://github.com/openedx/openedx-app-android/assets/1063094/28b55327-05d5-49a9-9d17-03e2378393ac

### CODE REVIEW

**Review by volodymyr-chekyrta — DISMISSED** [review]

**Comment by omerhabib26** [comment]
I'm currently reviewing this

**Review by omerhabib26 — CHANGES_REQUESTED** [review]
Hey @k1rill, Everything is working fine as expected only some minor nits. 
Also, I have a couple of inquiries:
1- When altering the app or device language during runtime, the URLs do not undergo any modification or update. Is this intended behavior?
2- Regarding the End User License Agreement (EULA), there appears to be no associated implementation. Should we display it on the Sign-In Page, similar to how it is presented in the edX App?
cc: @moiz994 @marcotuts @volodymyr-chekyrta

**Comment by moiz994** [comment]
> Hey @k1rill, Everything is working fine as expected only some minor nits. Also, I have a couple of inquiries: 1- When altering the app or device language during runtime, the URLs do not undergo any modification or update. Is this intended behavior? 2- Regarding the End User License Agreement (EULA), there appears to be no associated implementation. Should we display it on the Sign-In Page, similar to how it is presented in the edX App? cc: @moiz994 @marcotuts @volodymyr-chekyrta

Regarding the EULA link, I need to sync with compliance and Product at 2U to confirm implementation. I can create a separate ticket for this and we can do it in a separate PR.

File: core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
     <string name="core_error_unknown_error">Something went wrong</string>
     <string name="core_error_try_again">Try again</string>
     <string name="core_privacy_policy">Privacy policy</string>
+    <string name="core_cookie_policy" translatable="false">Cookie policy</string>
```
[the remark this answers is not in the document]
Comment (k1rill) [reply core/src/main/res/values/strings.xml:11]: After conversation we decided to keep this as is

**Comment by omerhabib26** [comment]
Just need to update the following string to `Sentance Case` else LGTM 🚀 
Screenshot 2024-01-04 at 2 10 00 AM

**Review by omerhabib26 — DISMISSED** [review]
Only a minor nit

**Review by volodymyr-chekyrta — APPROVED** [review]

**Review by omerhabib26 — APPROVED** [review]

**Pull request merged**